akka-docs/rst/java/code/docs/actor/ActorDocTest.java

```diff
@@ -469,7 +470,7 @@ public void creatingActorWithSystemActorOf() {
     //#system-actorOf
     // ActorSystem is a heavy object: create only one per application
     final ActorSystem system = ActorSystem.create("MySystem", config);
-    final ActorRef myActor = system.actorOf(Props.create(MyActor.class), "myactor");
+    final ActorRef myActor = system.actorOf(Props.create(MyJavaActor.class), "myactor");
```

Comment:
Was a conflict since the docs.actor package also exists in the scala samples. Would having a `docs.java.actor` be ok?